rmtk/plotting/damage_dist/parse_damage_dist.py

```diff
@@ -10,7 +10,7 @@
 from lxml import etree
 from collections import OrderedDict
 
-xmlNRML='{http://openquake.org/xmlns/nrml/0.4}'
+xmlNRML='{http://openquake.org/xmlns/nrml/0.5}'
```

Comment:
Could you check that this has also been updated for the **plot_damage_dist.py** script?